input/docs/handbook/commands/binding-commands.md

```diff
@@ -1,63 +1,63 @@
 # Binding
 
-Once you have created a command and exposed it from your view model, the next logical step is to consume it from your view. The most common means of achieving this is via the `BindCommand` method. This method - of which there are several overloads - is responsible for tying any source `ICommand` to a target control. Typical usage looks like this:
+Commands exposed in the view model are typically meant to be consumed by the view through its `BindCommand` method. `BindCommand` and its overloads' job is to tie `ICommand`s in the view model to controls in the view. Here's an example:
```

Comment:
Wonder if 

The `BindCommand` set of methods tie `ICommand` derived commands from the view model to the controls in the view.